packages/lake/src/components/QuickActions.tsx

```diff
@@ -40,39 +47,48 @@ export type QuickAction = {
 
 type Props = {
   actions: QuickAction[];
+  tooltipDisabled?: boolean;
+  tooltipText?: string;
 };
 
-export const QuickActions = ({ actions }: Props) => {
+export const QuickActions = ({ actions, tooltipDisabled = false, tooltipText }: Props) => {
   return (
     <View style={styles.container}>
       {actions.map((action, index) => (
-        <Pressable
+        <LakeTooltip
+          content={tooltipText}
+          placement="bottom"
           key={index}
-          onPress={action.onPress}
-          style={styles.action}
-          disabled={action.isLoading === true}
+          disabled={tooltipDisabled}
```

Comment:
maybe we can also disabled it if tooltipText is undefined or empty.  
`disabled={tooltipDisabled} || isNullishOrEmpty(tooltipText)`  
It will avoid an empty tooltip in case `tooltipText` is not defined.  
What do you think?

Reply:
Yes good idea!